lib/mindwendel_web/live/brainstorming_live/show.html.heex

```diff
@@ -21,100 +21,17 @@
-        <% end %>
-      <% end %>
-
-      <%= live_patch to: Routes.brainstorming_show_path(@socket, :share, @brainstorming), class: "btn btn-secondary m-1", title: gettext("Share") do %>
-        <i class="bi-share-fill"></i>
-      <% end %>
-
-      <%= if @brainstorming.option_show_link_to_settings do %>
-        <%= link to: Routes.admin_brainstorming_edit_path(@socket, :edit, @brainstorming.admin_url_id), class: "btn btn-secondary m-1" do %>
-          <i class="bi-gear-fill"></i>
-        <% end %>
-      <% end %>
-
-      <div
-        class="deletion-date-hint btn btn-outline-secondary m-1"
-        data-bs-toggle="tooltip"
-        data-bs-placement="auto"
-        data-bs-custom-class="deletion-date-hint-tooltip"
-        data-bs-title={brainstorming_available_until_full_text(@brainstorming)}
-      >
-        <i class="bi-calendar-x"></i> <%= brainstorming_available_until(@brainstorming) %>
-      </div>
-    </div>
-
-    <%= if @live_action in [:edit] do %>
-      <%= live_modal(MindwendelWeb.BrainstormingLive.FormComponent,
-        id: @brainstorming.id,
-        title: @page_title,
-        action: @live_action,
-        brainstorming: @brainstorming,
-        return_to: Routes.brainstorming_show_path(@socket, :show, @brainstorming)
-      ) %>
-    <% end %>
-
-    <%= if @live_action in [:new_idea] do %>
-      <%= live_modal(MindwendelWeb.IdeaLive.FormComponent,
-        id: :new,
-        title: gettext("New idea"),
-        action: :new,
-        brainstorming: @brainstorming,
-        current_user: @current_user,
-        idea: @idea,
-        return_to: Routes.brainstorming_show_path(@socket, :show, @brainstorming)
-      ) %>
-    <% end %>
-
-    <%= if @live_action in [:edit_idea] do %>
-      <%= live_modal(MindwendelWeb.IdeaLive.FormComponent,
-        id: :update,
-        title: gettext("Update idea"),
-        action: :update,
-        brainstorming: @brainstorming,
-        current_user: @current_user,
-        idea: @idea,
-        return_to: Routes.brainstorming_show_path(@socket, :show, @brainstorming)
-      ) %>
-    <% end %>
-
-    <%= if @live_action in [:share] do %>
-      <%= live_modal(MindwendelWeb.BrainstormingLive.ShareComponent,
-        id: :share,
-        title: gettext("Share brainstorming"),
-        action: :share,
-        brainstorming: @brainstorming,
-        current_user: @current_user,
-        uri: Routes.brainstorming_show_url(@socket, :show, @brainstorming),
-        modal_size: "modal-lg",
-        return_to: Routes.brainstorming_show_path(@socket, :show, @brainstorming)
-      ) %>
-    <% end %>
-
-    <%= live_component(MindwendelWeb.LabelLive.CaptionsComponent,
-      ideas: @ideas,
-      brainstorming: @brainstorming
-    ) %>
 
-    <%= live_component(MindwendelWeb.IdeaLive.IndexComponent,
-      ideas: @ideas,
-      brainstorming: @brainstorming,
-      id: @brainstorming.id,
-      current_user: @current_user
-    ) %>
```

Comment:
Jannik so: Lol let's see how long Tobi takes until he finds out I removed the ideas from rendering in a brainstorming :angel:

Reply:
😂 I sometimes start with just removing stuff and introducing it back in one by one. the problem currently is that there are so many errors that you would need to remove basically everything 😂